scripts/get_usernames.py

```diff
@@ -0,0 +1,20 @@
+import csv
+import json
+
+def make_json(csvFilePath, jsonFilePath):
+	data = []
+
+	with open(csvFilePath, encoding='utf=8') as csvf:
+		csvReader = csv.DictReader(csvf)
+
+		for row in csvReader:
+			data.append(row['login'])
```

Comment:
Iterating through `csvReader` should not be in the `with` clause
```suggestion
	with open(csvFilePath, encoding='utf=8') as csvf:
		csvReader = csv.DictReader(csvf)

	for row in csvReader:
		data.append(row['login'])
```